src/css/styles.css

```diff
@@ -116,19 +151,26 @@ a:hover {
   padding: 0 70px;
 }
 
-@media only screen and (max-width: 767px) {
+.text-group h2 {
+  margin: 0 auto;
+  text-align: center;
 }
-
+@media only screen and (max-width: 1300px) {
+  .section {
+    padding: 90px 100px;
+  }
+}
+@media only screen and (max-width: 767px) {}
 @media only screen and (min-width: 768px) and (max-width: 991px) {
   body {
     font-size: 18px;
     line-height: 1.5;
   }
 
+  .subhead,
```

Comment:
just curious, is this based on some style guide? i dont have a strong opinion about the order

Reply:
I believe this is prettier at work. I didn't reorder, but i like it.